setup/package2.py

```diff
@@ -0,0 +1,569 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import argparse
+import logging
+import os
+import pexpect
+import shutil
+import signal
+import subprocess
+import sys
+import tempfile
+import time
+import traceback
+try:
+    from xmlrpc import client as xmlrpclib
+except ImportError:
+    import xmlrpclib
```

Comment:
Probably not useful if this only runs on P3.

Reply:
ok, I remove it